source/js/components/sticky-nav.js

```diff
@@ -24,10 +24,33 @@ var StickyNav = React.createClass({
       active: window.location.hash.replace(`#`, ``) || ``
     };
   },
+  componentDidMount: function() {
+    document.addEventListener(`scroll`, this.onScroll);
+  },
+  componentWillUnmount: function() {
+    document.removeEventListener(`scroll`, this.onScroll);
+  },
+  onScroll: function() {
```

Comment:
The wrapper for this is main.js. Not a great place to put something that changes the state of another component. Especially if it's not a react component.

I am going to try moving out the scrolling bits a bit to be a bit more separate from the sticky concept. Hopefully that's enough?